src/Engine/RadiumEngine.hpp

```diff
@@ -171,6 +171,83 @@ class RA_ENGINE_API RadiumEngine
      */
     Core::Aabb computeSceneAabb() const;
 
+    /// \name Time Management.
+    /// {
+
+    /**
+     * Toggles effective time flow or constant time flow.
```

Comment:
Even if experienced users will understand this comment, give a little explanation of what is "effective time" or "constant time" and state the relation of these with the "RealTime" name of the method.